### PR TITLE
Add email about discord roles and supervisor role

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Fixes https://github.com/srobo/tasks/issues/713

- Remind teams not on discord to join
- Remind teams without a leader to join or flag their username
- Mention some definitions of the role of the supervisor (will end up in the docs eventually).

### CODE REVIEW

**Review by Adimote — DISMISSED** [review]

**Comment by antoinepetty** [comment]
I think we should switch the focus to the status update and have the subject more actionable like "How is your team getting on?"

File: SR2022/2022-02-10-discord-roles.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2022 missing supervisors in Discord
+subject: How is yor team getting on?
```
Comment (antoinepetty) [inline SR2022/2022-02-10-discord-roles.md:3]:
```suggestion
subject: How is your team getting on?
```

**Comment by PeterJCLaw** [comment]
Suggest closing this given that we've since had separate contact with most teams and this seems less relevant at this stage?

**Comment by antoinepetty** [comment]
Agree

**Pull request closed without merging**